datacenter/dtr/2.1/guides/install/upgrade.md

```diff
@@ -13,6 +13,12 @@ There is no downtime when upgrading an highly-available DTR cluster. If your
 DTR deployment has a single replica, schedule the upgrade to take place outside
 business peak hours to ensure the impact on your business is close to none.
 
+> **Warning**
```

Comment:
Can you link to `/datacenter/dtr/2.1/guides/high-availability/backups-and-disaster-recovery/`. It provides more guidance than the reference?